src/python_testing/TC_RVCOPSTATE_2_4.py

```diff
@@ -0,0 +1,174 @@
+        return "MopCleaningPadMissing(0x47)"
+
+
+class TC_RVCOPSTATE_2_3(MatterBaseTest):
+    def __init__(self, *args):
+        super().__init__(args)
+        self.endpoint = self.matter_test_config.global_test_params['PIXIT_ENDPOINT']
+
+    async def read_mod_attribute_expect_success(self, endpoint, attribute):
+        cluster = Clusters.Objects.RvcOperationalState
+        return await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attribute)
+
+    async def send_go_home_cmd(self) -> Clusters.Objects.RvcOperationalState.Commands.GoHome:
+        ret = await self.send_single_cmd(cmd=Clusters.Objects.RvcOperationalState.Commands.GoHome(), endpoint=self.endpoint)
+        asserts.assert_true(type_matches(ret, Clusters.Objects.RvcOperationalState.Commands.OperationalCommandResponse),
+                            "Unexpected return type for GoHome")
+        return ret
+
+    # Sends the GoHome command and checks that the returned error matches the expected_error
+    async def send_go_home_cmd_with_check(self, step_number, expected_error):
+        self.print_step(step_number, "Send GoHome command")
+        ret = self.send_go_home_cmd()
+        asserts.assert_equal(ret.commandResponseState.errorStateID, expected_error,
+                             "errorStateID(%s) should be %s" % ret.commandResponseState.errorStateID,
+                             error_enum_to_text(expected_error))
+
+    # Prints the step number, reads the operational state attribute and checks if it matches with expected_state
+    async def read_operational_state_with_check(self, step_number, expected_state):
+        self.print_step(step_number, "Read OperationalState")
+        operational_state = await self.read_mod_attribute_expect_success(
+            endpoint=self.endpoint, attribute=Clusters.RvcOperationalState.Attributes.OperationalState)
+        logging.info("OperationalState: %s" % operational_state)
+        asserts.assert_equal(operational_state, expected_state,
+                             "OperationalState(%s) should be %s" % operational_state, state_enum_to_text(expected_state))
+
+    # Prints the instruction and waits for a user input to continue
+    def print_instruction(self, step_number, instruction):
+        self.print_step(step_number, instruction)
+        input("Press Enter when done.\n")
+
+
+    @async_test_body
+    async def test_TC_RVCOPSTATE_2_4(self):
+
+        asserts.assert_true('PIXIT_ENDPOINT' in self.matter_test_config.global_test_params,
+                            "PIXIT_ENDPOINT must be included on the command line in "
+                            "the --int-arg flag as PIXIT_ENDPOINT:<endpoint>")
+
+        asserts.assert_true(self.check_pics("RVCOPSTATE.S.A0004"), "RVCOPSTATE.S.A0004 must be supported")
+        asserts.assert_true(self.check_pics("RVCOPSTATE.S.C80.Rsp"), "RVCOPSTATE.S.C80.Rsp must be supported")
```

Comment:
```suggestion
        asserts.assert_true(self.check_pics("RVCOPSTATE.S.C04.Rsp"), "RVCOPSTATE.S.C04.Rsp must be supported")
```

I assume this is the PICS for the GoHome command. BTW, the test plan PR (3750) has a bug re. this, and I'll fix it shortly.

Reply:
Is `C04` meant to match with the [command ID as described in the spec](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/a0686eddba3b04212c3b0f8ec224e8a13ae6dce2/src/app_clusters/OperationalState_RVC.adoc?plain=1#L198)? If so, it should be `C08`. If not, can you explain where the `C04` comes from?

Reply:
@hicklin Sorry, that was a mistake. I edited my suggestion. The command id is 0x80 so 128 decimal.